themes/edx.org/lms/templates/footer.html

```diff
@@ -0,0 +1,83 @@
+## mako
+<%!
+  from django.utils.translation import ugettext as _
+  from branding.api import get_footer
+%>
+<% footer = get_footer(is_secure=is_secure) %>
+<%namespace name='static' file='static_content.html'/>
+
+## WARNING: These files are specific to edx.org and are not used in installations outside of that domain. Open edX users will want to use the file "footer.html" for any changes or overrides.
```

Comment:
I'd rather not modify this file in this pull request. I'm simply copying the `footer-edx-v3.html` file into the edx.org theme directory. Modifications to this file can happen in a future pull request.